tests/python_package_test/test_dask.py

```diff
@@ -13,6 +16,7 @@
 if not lgb.compat.DASK_INSTALLED:
     pytest.skip('Dask is not installed', allow_module_level=True)
 
+import cloudpickle
```

Comment:
I believe `cloudpickle` should be added here as we importing it directly, we shouldn't rely on that it is a second-order dependency of some package.
https://github.com/microsoft/LightGBM/blob/ea8e47ea24fb942155f16da6720b32de976d5731/.ci/test.sh#L93

Reply:
sure, done in https://github.com/microsoft/LightGBM/pull/3883/commits/344376bee1a7a3dbe97f7078a782d091d1084c30